src/main/java/com/example/model/controllers/ExceptionController.java

```diff
@@ -0,0 +1,33 @@
+package com.example.model.controllers;
+
+
+import com.example.model.exceptions.RecordNotFoundException;
+import com.example.model.exceptions.TelevisionNameTooLongException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionController {
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity<String> exception(RecordNotFoundException exception) {
+
+        return new ResponseEntity<>("Deze tv is niet gevonden", HttpStatus.NOT_FOUND);
```

Comment:
Je laat met deze String (en de Strings in je andere exceptionhandlers) goed zien waar de verschillende exeptions voor gebruikt worden. Weet ook dat wanneer een exception in verschillende situaties gebruikt kan worden, je de String (het bericht) ook al bij het opgooien van de message kan meegeven (`throw new ...Exception("message hier")`) en het hier kan aanroepen. In plaats van een hardcoded String, kun je hiet dan `exception.getMessage()` doen.